components/molecule/tabs/src/components/MoleculeTabs.js

```diff
@@ -36,26 +38,39 @@ const MoleculeTabs = ({
         autoScrollIntoView,
         isIntersecting,
         numTab,
+        id,
         onChange
       })
     })
 
   const activeTabContent = childrenArray.reduce((activeContent, child) => {
     if (child) {
-      const {children: childrenChild, active} = child.props
-      return active ? childrenChild : activeContent
+      const {children: childrenChild, active, numTab} = child.props
+
+      return active ? (
+        <div className={CLASS_CONTENT} id={`${id}-${numTab}`} role="tabpanel">
+          {childrenChild}
+        </div>
+      ) : (
+        activeContent
+      )
```

Comment:
`return (<div className={CLASS_CONTENT} id={`${id}-${numTab}`} role="tabpanel">{active ? childrenChild : activeContent}</div>)`

Reply:
Nope, it's not well. Your solution add the active tab content inside another active tab content

![image](https://user-images.githubusercontent.com/3933098/194880297-f283253a-12f5-4528-9fae-f53cd9e808cd.png)

Reply:
```js
if(active) {
  return (
    <div className={CLASS_CONTENT} id={`${id}-${numTab}`} role="tabpanel">
      {childrenChild}
    </div>
  )
}
```